docs/sla.md

```diff
@@ -0,0 +1,93 @@
+# Service Level Agreement(SLA)
+
+Service Level Agreement is available to Premium Customers only.
+
+
+## Definitions
+
+* "Self-service Console" provides access to a self-service console at https://console.cdnetworks.com. The console provides a way to configure the service through the Web UI or API. 
+* "Self-service Documentation" is a self-service documentation portal at https://docs.cdnetworks.com and can be accessed at any time.
```

Comment:
Again, defining a term by repeating is not helpful.
I would just say something like "Self-service documentation" is our online help available at https://docs.cdnetworks.com.

Reply:
I agree with Burton's suggestion.